Mathlib/GroupTheory/GroupAction/Basic.lean

```diff
@@ -274,18 +274,13 @@ end MulAction
 namespace AddAction
 variable {G α : Type*} [AddGroup G] [AddAction G α]
 
-@[simp]
-theorem _root_.AddAut.conj_symm_apply'  {G : Type*} [AddGroup G] (g h : G) :
```

Comment:
I guess this should be deprecated

Reply:
I'll delete that one. It was a temporary stuff to check what happens! PR #23978 makes it useless.